src/app/portlets/dot-edit-page/layout/dot-edit-layout-grid/dot-edit-layout-grid.component.ts

```diff
@@ -0,0 +1,131 @@
+import { Component, OnInit } from '@angular/core';
+import { NgGridConfig, NgGridItemConfig } from 'angular2-grid';
+import _ from 'lodash';
+
+//Final Object need to be defined.
+interface DotLayoutContainer {
+    id: string;
+    config: NgGridItemConfig;
+}
+
+@Component({
+    selector: 'dot-edit-layout-grid',
+    templateUrl: './dot-edit-layout-grid.component.html',
+    styleUrls: ['./dot-edit-layout-grid.component.scss']
+})
+export class DotEditLayoutGridComponent implements OnInit {
```

Comment:
You need to add an ```@Input()``` here so we can pass the model from the parent container (aka from the server```